dpgen2/exploration/render/traj_render_lammps.py

```diff
@@ -63,6 +63,8 @@ def get_confs(
         id_selected: List[List[int]],
         type_map: Optional[List[str]] = None,
         conf_filters: Optional["ConfFilters"] = None,
+        use_ele_temp: int = 0,
+        ele_temp: Optional[List[float]] = None,
```

Comment:
perhaps let the `traj_render` to handle extra output files, not the `conf_selector`.